package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jupiterone/integration-template",
```

Comment:
Need to update this to `@jupiterone/graph-databricks`

Reply:
Ouch, good find - will do!